### DISCUSSION TITLE
WIP: Ready Player Me Avatar Animation for Godot

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [godotengine/godot-proposals#6908] [3D]
[**ReadyPlayerMe-Godot-Test**](https://github.com/srcnalt/ReadyPlayerMe-Godot-Test/issues/1#issue-1713856035)

- [RPM-Face-Tracing in Godot using C#](https://github.com/srcnalt/rpm-face-tracking/issues/1)

**Comment by dukemagus** [comment]
Maybe as a plugin, but linking branded commercial models and API into the main project could be received poorly.
